README.md

```diff
@@ -157,6 +157,51 @@ process.env.APPLICATIONINSIGHTS_CONFIGURATION_FILE = "C:/applicationinsights/con
+| setAutoCollectIncomingRequestAzureFunctions | Auto collection of Azure Functions is not supported by the shim. Please migrate to the distro for support. |
+
+The following configurations are set using either environment variables, setting them in the `applicationinsights.json` file or by calling `appInsights.defaultClient.config.<CONFIG_SETTING_VALUE>;`.
+
+|Property               |Support Status                                                         |
+|-----------------------|-----------------------------------------------------------------------|
+| instrumentationKey & endpointUrl | Not supported by the shim or the distro. Please migrate to using the connectionString. |
+| maxBatchSize | Not supported by the shim but can be configured by using OpenTelemetry SpanProcessors in the distro. |
+| disableAppInsights | Not supported by the shim. Disabling telemetry export is possible via the distro using OpenTelemetry.|
+| correlationIdRetryIntervalMs | Not supported by either the shim or the distro as the value is not configurable in OpenTelemetry.|
+| ignoreLegacyHeaders | Legacy headers in outgoing requests are not supported in the shim or the distro. Therefore they will always be disabled. |
+| distributedTracingMode | Distributed tracing mode is always set to AI_AND_W3C. AI only tracing mode is not supported in the shim or the distro.|
+| enableLoggerErrorToTrace | Not supported in the shim or the distro as all errors will be logged as exceptions in both. |
+| enableAutoCollectHeartbeat | Not supported in the shim or the distro. |
+| enableAutoDependencyCorrelation | Cannot disable dependency correlation in either the shim or the distro. Dependency correlation will always be enabled and therefore this setting is always true. |
+| enableAutoCollectIncomingRequestAzureFunctions | Not supported by the shim. Migrate to the distro to use auto collection of Azure Functions. |
+| enableUseAsyncHooks | Using async hooks is not supported by the shim or the distro as it is not supported by OpenTelemetry. |
+| enableResendInterval | Not supported by the shim. It is possible to configure the interval between exports via OpenTelemetry span processors, but not specifically cached events. The @azure/monitor-opentelemetry-exporter uses a resend interval of one minute. |
+| enableMaxBytesOnDisk | Not supported by the shim. And not supported to be changed in the distro. The @azure/monitor-opentelemetry-exporter sets this value at 50MB. |
+| noHttpAgentKeepAlive | Not supported in the shim or the distro. |
+| httpAgent/httpsAgent | Not supported in the shim or the distro. |
+| enableWebInstrumentation | Not currently supported in the shim or the distro. |
+| webInstrumentationConnectionString | Not supported in the shim or the distro. |
+| webInstrumentationConfig | Not currently supported by the shim or the distro. |
+
```

Comment:
We are still lacking TelemetryClient documentation,, if you are planning to follow up later please create a task so we do not forget

Reply:
Added unsupported TelemetryClient fields here, as well as noting that the required fields for `trackPageView` are slightly different.